src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -151,12 +131,7 @@ const SetAclPermissionForm = () => {
           />
 
           <FormControl fullWidth sx={{ marginTop: '2rem' }}>
-            <Button
-              variant="contained"
-              disabled={permissionState.podUrlToSetPermissionsTo === podUrl || state.processing}
-              type="submit"
-              color="primary"
-            >
+            <Button variant="contained" disabled={processing} type="submit" color="primary">
```

Comment:
The previous disabling had the button also disabled it when the user put in their own podUrl - so they couldn't try to change permissions of their own pod.   Think we need to add this back in, unless you found another way to address that possibility.